style.css

```diff
@@ -9,16 +9,27 @@
 */
 
 body {
-	margin: 80px;
-	background: #fff;
+   margin: 40px;
 	font-family: 'Poiret One', cursive;
```

Comment:
This one is going to be quite a task, on my sublime everything is aligned perfectly. It might be the lines that had the cursor on the beginning of the line and I must have tabbed it to align it and github recorded the tab...